src/business_logic/fact_state/in_memory_state_reader.rs

```diff
@@ -7,17 +7,19 @@ use crate::{
     utils::Address,
 };
 use felt::Felt;
+use getset::MutGetters;
 use std::collections::HashMap;
 
-#[derive(Clone, Debug)]
-pub(crate) struct InMemoryStateReader {
+#[derive(Clone, Debug, MutGetters)]
+pub struct InMemoryStateReader {
     pub(crate) ffc: DictStorage,
+    #[getset(get_mut = "pub")]
```

Comment:
Why do we need this? @azteca1998

Reply:
The library reduces get/setter boilerplate. As for the methods themselves, we need them at the integration tests.